scripts/lib/api/updateLinks.ts

```diff
@@ -25,6 +25,26 @@ import { removePart, removePrefix } from "../stringUtils";
 import { HtmlToMdResultWithUrl } from "./HtmlToMdResult";
 import { remarkStringifyOptions } from "./commonParserConfig";
 import { Link } from "../sharedTypes";
+import { ObjectsInv } from "./objectsInv";
+import { transformSpecialCaseUrl } from "./specialCaseResults";
+
+/**
+ * Anchors generated from markdown headings are lower case
```

Comment:
It'd be helpful to add a `"but ..."` explaining why this function is necessary, e.g.:

```
but Sphinx was using uppercase unlike our platform
```

Reply:
Added more context in 0131c9da509ccd40f96cc3194fc9b81959f4f021